.github/workflows/ci.yml

```diff
@@ -17,20 +16,14 @@ jobs:
     strategy:
       matrix:
         java-version:
-          - 8
           - 11
-    env:
-      HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
-      HEROKU_API_USER: ${{ secrets.HEROKU_API_USER }}
```

Comment:
Are these going to be unset in the GitHub dashboard for now (x2 given Dependabot section too), or are the integration tests being added soon enough that it's worth leaving them set?